### DISCUSSION TITLE
Plan for Phone Auth

### DISCUSSION THREAD

**Discussion by ilhamwahabi** [supabase/supabase#1022] [Questions] [answered]
Hi, from my exploration looks like authentication already support email-password, oauth, and magic-link.

Any plan for phone auth? I mean login with phone number and using OTP. One of the pain using phone auth in Firebase is we can't modify the message so if this planned it would be huge win.

Thank you.

**Comment by silentworks** [comment, marked as answer]
Phone auth is now available. You can read more about it in this blog post https://supabase.io/blog/2021/07/28/supabase-auth-passwordless-sms-login
